_data/providers.json

```
@@ -161,6 +161,11 @@
       "url": "https://eendhosting.nl/",
       "description_template": "default"
     },
+    {
+      "name": "ElfIDC",
+      "url": "https://www.elfidc.com/",
+      "description": "When placing an order, you can choose Paper/Spigot + Geyser, he will automatically install the Geyser plugin, and the player will connect through the Java IP and port, which is built-in."
```

```suggestion
      "description": "When placing an order, choose Paper/Spigot + Geyser, which will automatically install the Geyser plugin. Connect with the Java server's IP and port. Note: This provider only provides service in China."
```
I'd change the phrasing a bit - additionally, we historically added a note for country specific hosts (e.g. Russia). If this isn't the case, feel free to correct it

---

They offer nodes that are based in Hong Kong, Los Angeles, and Japan: http://node.elfidc.com/

---

I changed it and wish they can active